frontend/src/helpers/transformations.js

```diff
@@ -38,3 +38,77 @@ export function optionsFromIds(addressBook, ids) {
   }
   return options;
 }
+
+// Returns an array of objects (in the current language) with the following properties
+// id (required): number in order indexed at 0
+// name (required): string displayed in the tree
+// groups: array of numbers of the children
+// parent: number id of the parent
+// level (required): number representing the level of the tree you are in
+export const processTreeContent = (
+  currentLanguage,
+  treeViewContent_en,
+  treeViewContent_fr,
+  treeType
+) => {
+  const treeContent = currentLanguage === "en" ? treeViewContent_en : treeViewContent_fr;
```

Comment:
Can you please import `LANGUAGES` and use it like that instead?
`currentLanguage === LANGUAGES.english`

Reply:
Will do.